lib/ecto/adapters/clickhouse.ex

```diff
@@ -144,8 +146,30 @@ defmodule Ecto.Adapters.ClickHouse do
   defdelegate structure_dump(default, config), to: Ecto.Adapters.ClickHouse.Structure
 
   @impl Ecto.Adapter.Structure
-  def structure_load(_default, _config) do
-    raise "not implemented"
+  def structure_load(default, config) do
```

Comment:
Can we maybe move it to `Ecto.Adapters.ClickHouse.Structure`?

Reply:
```elixir
defdelegate structure_load(default, config), to: Ecto.Adapters.ClickHouse.Structure
```

Reply:
Done